dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -359,7 +359,9 @@ internal open class RumViewScope(
         val rumContext = getRumContext()
 
         val updatedAttributes = addExtraAttributes(event.attributes)
-        val isFatal = updatedAttributes.remove(RumAttributes.INTERNAL_ERROR_IS_CRASH) as? Boolean
+        val isFatal = updatedAttributes.remove(RumAttributes.INTERNAL_ERROR_IS_CRASH) as? Boolean == true || event.isFatal
+        if (crashCount > 0 && isFatal) return // if a cross-platform crash was already reported, do not send its native version
```

Comment:
This could work only if we are sure there's not going to be another fatal crash reported after this.

Reply:
If the crash is fatal, none could be reported after.